packages/dd-trace/test/llmobs/sdk/index.spec.js

```diff
@@ -1258,6 +1258,38 @@ describe('sdk', () => {
         value: 'it is super toxic!'
       }), { message: 'value must be a boolean for a boolean metric' })
     })
+
+    it('adds source:otel tag when DD_TRACE_OTEL_ENABLED is set', () => {
+      process.env.DD_TRACE_OTEL_ENABLED = 'true'
```

Comment:
i would just move these into `before` and `after` hooks, like

```javascript
describe('with DD_TRACE_OTEL_ENABLED set', () => {
  before(() => {
      process.env.DD_TRACE_OTEL_ENABLED = true     
  })

  after(() => {
      delete process.env.DD_TRACE_OTEL_ENABLED
  })

  it('adds a source:otel tag', () => { ... })
})
```

in case something goes wrong with the test, we can clean up the env after (sorta like pytest fixtures)